application/models/Question.php

```diff
@@ -1525,4 +1526,15 @@ public function getScaledSubquestions()
         }
         return $results;
     }
+
+    /**
+     * Get the question theme name
+     *
+     * @return string
```

Comment:
Myeh. This should probably return question theme object, no?

Later it should be possible to do `$question->question_theme`, too. But impossible now, since there's no database column for it. :(

Reply:
Yes,  having the question_theme as attribute is important. Maybe we can add a story for that?
We can always update the methods later (or add new ones).

Reply:
I'll add it.

Reply:
Still, can you change this to getQuestionThemeName? To be clear it's not a domain object you get.